src/or/main.c

```diff
@@ -1458,6 +1458,55 @@ initialize_periodic_events_cb(evutil_socket_t fd, short events, void *data)
   }
 }
 
+/* Helper function: Set up periodic_events[] that only matches the given
```

Comment:
Documentation should be a doxygen comment, and should explain what "matches" means.